package.json

```diff
@@ -33,33 +33,35 @@
     "!lib/**/*.map"
   ],
   "dependencies": {
-    "@salesforce/core": "^2.31.0",
-    "@salesforce/ts-types": "^1.2.1",
+    "@salesforce/core": "^3.15.5",
+    "@salesforce/ts-types": "^1.5.20",
     "applicationinsights": "^1.4.0",
     "axios": "^0.27.2"
   },
   "devDependencies": {
-    "@salesforce/dev-config": "^2.0.0",
-    "@salesforce/dev-scripts": "^0.9.18",
+    "@salesforce/dev-config": "^3.0.1",
+    "@salesforce/dev-scripts": "^2.0.2",
```

Comment:
The `prepare` script should be set to `sf-install` to make dev-scripts work (it's missing the husky folder).